twenty-first/src/shared_math/tip5.rs

```diff
@@ -909,6 +912,37 @@ mod tip5_tests {
         );
     }
 
+    #[test]
+    fn hash_var_len_equivalence_corner_cases() {
+        // Verify that `hash_var_len` is equivalent
+        // to pad + repeated absorb + squeeze once
+        for preimage_length in [0, 1, 2, 3, 4, 5, 6, 9, 10, 11] {
```

Comment:
might as well do
```suggestion
        for preimage_length in 0..=11 {
```
or
```suggestion
        for preimage_length in (0..=6).chain(9..=11) {
```
if leaving out 7 and 8 is important. The long list of integers is time consuming to parse imo.